src/todolist.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable linebreak-style */
+import Drag from '../icon/drag.png';
+
+const completed = false;
+let i = 0;
+export default class ToDoList {
+  constructor() {
+    this.todolist = [];
+    this.todolist = this.getList();
+  }
+
+  getList = () => {
+    if (!this.checkList()) {
+      return JSON.parse(localStorage.getItem('list'));
+    }
+    return this.todolist;
+  }
+
+  checkList = () => (localStorage.getItem('list') === null);
+
+  updateList = () => {
+    localStorage.setItem('list', JSON.stringify(this.todolist));
+    setTimeout(() => {
+      window.location.reload();
+    }, 100);
+  }
+
+  displayList = () => {
+    i = 0;
+    const list = document.querySelector('ul');
+    Object.values(this.todolist).forEach((val) => {
+      list.innerHTML += `<li class="list-item" draggable="true">
+                            <span>
+                              <input id='checker${i}' class='checker' type='checkbox'>
+                              <input id="lists${i}" class="lists" type="text" value="${val.description}">
+                            </span>
+                            <img class="drag" draggable="false" id='drag${i}' src='${Drag}'>
+                          </li>`;
+      i += 1;
+    });
+  }
```

Comment:
I really like the order you used in your code, everything looks very neat and clean.

In this code you use a variable i to control list printing and it is a very good solution. However, the function you are using "forEach()" also has the ability to give you the index without declaring an additional variable.

you could use it this way to make your code look more professional:
`  forEach( (val, index) => { /* ... */ })`

Reply:
This was intentional. If I used index for the foreach, It would have incremented on the next iteration. But by using variable, I can increment when I wanted to increement.